internal/cli/deployment_list.go

```diff
@@ -211,27 +201,33 @@ func (c *DeploymentListCommand) Run(args []string) int {
 
 			if user, ok := b.Labels["common/user"]; ok {
 				details = append(details, "user:"+user)
-			} else if user, ok := b.Preload.Build.Labels["common/user"]; ok {
-				details = append(details, "build-user:"+user)
-			}
+			} else if deployBundle.Build != nil {
+				build := deployBundle.Build
+				// preload labels have been set, safe to use them
 
-			if bp, ok := b.Preload.Build.Labels["common/languages"]; ok {
-				details = append(details, niceLanguages(bp))
-			}
+				if user, ok := build.Labels["common/user"]; ok {
```

Comment:
I think we end up appending this twice? We're also looking at this label on 202.

Reply:
Although one is user and one is build-user 🤔 not sure the difference.

Reply:
I was just replicating the old logic, but with more nil checking 🙃. It looks to me like we'll either set that `user` field from the top-level label's `common/user`, or from the build labels `common/user` if the former isn't present. Either way, it should be the same as before.

Reply:
I see! Should the details here just be `user:` then if it's the same?

Reply:
I don't think it'll necessarily be the same - if we don't find user on top level labels and resort to pulling it off the build, I think it makes sense to say we're showing the `build-user`.